### DISCUSSION TITLE
Cannot cancel matrix job using reusable workflow stuck in pending state

### DISCUSSION THREAD

**Discussion by thered0ne** [community/community#125017] [Actions]
### Select Topic Area

Bug

### Body

I have a Git Actions Workflow where the triggering is
on:
workflow_dispatch:

This has two jobs. The second one is a matrix which uses another reusable workflow. 
The issue is that I have 3 workflows runs stuck on the Matrix job reporting `Waiting for pending jobs` for 4 days now.
I have tried to cancel the workflow from `GUI, Rest API and gh cli` but this does not have any effect.
This exact job had successful runs before. Runners used for the matrix job are self-hosted and currently they are successfully running other workflows.

How could I cancel these workflow runs and further debug the issue ?

**Comment by grandmas-favorite** [comment]
Hi @thered0ne!

This issue was likely caused by an [incident](https://www.githubstatus.com/incidents/nj47vccwm2zj) reported this morning that has since been updated, and Actions should be operating normally now.
